app/assets/stylesheets/_tahi.scss

```diff
@@ -56,7 +56,7 @@ textarea { resize: none; }
 
 
 .matte #tahi-container {
-  padding-top: 135px;
+  padding-top: 114px;
```

Comment:
Rule sets should be ordered as follows: @extends, @includes without @content, properties, @includes with @content, nested rule sets